Content.IntegrationTests/Tests/PostMapInitTest.cs

```diff
@@ -65,6 +65,7 @@ public sealed class PostMapInitTest
             "Saltern2",
             "Cluster2",
             "Atlas2"
```

Comment:
This is the source of the errors.
```suggestion
            "Atlas2",
```